src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -243,3 +285,5 @@ def test_WINNF_FT_S_FAD_2(self):
     }
     response = self._sas.Heartbeat(request)['heartbeatResponse'][0]
     self.assertEqual(response['response']['responseCode'] in (103, 500))
+    # shutdown the server
+    stopServer(self._sas_server)
```

Comment:
If the SAS fails one of the previous asserts, will the server be stopped? Or will it continue to run and cause trouble for the next test case? Can we use variable scoping to solve this problem?